plugin/pom.xml

```diff
@@ -51,6 +51,7 @@
     <argLine>-Djava.awt.headless=true -Xmx1024m -Djenkins.test.timeout=1000 --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED --add-opens java.base/java.util=ALL-UNNAMED --add-opens java.base/java.util.concurrent=ALL-UNNAMED</argLine>
 
     <prism-api.version>1.29.0-8</prism-api.version>
+    <jenkins.version>2.414.1</jenkins.version>
```

Comment:
gson-api need minimum 2.401.3. Not 2.414.1

Reply:
I had done tried both the comments.

In this case, I get the error:

```
[ERROR] Failed to execute goal org.jenkins-ci.tools:maven-hpi-plugin:3.50:validate-hpi (default-validate-hpi) on project warnings-ng: Dependency io.jenkins:configuration-as-code:jar:1714.v09593e830cfa requires Jenkins 2.414.1 or higher.
```